### DISCUSSION TITLE
iTwin Snapshot Geographical coordinates

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#707] [Q&A]
i set some dgn file to bim file by iTwin Snapshot ,but the dgn files not set Geographical coordinates,
so
1.set or change the Geographical coordinates(Latitude and longitude) to bim file by imodeljs program code?
2.the iTwin Snapshot have the config to default Geographical coordinates?

**Comment by mjunaid-khan** [comment]
Hi, it should work out of the box. Can you share some test files to Junaid.Khan@bentley.com or Jeff.Marker.@bentley.com. Thanks
